scripts/install_submitter.py

```diff
@@ -0,0 +1,768 @@
+            return True
+
+        try:
+            tree = ET.parse(xml_path)
+            root = tree.getroot()
+
+            # Find "python enable sandbox" key
+            # XML structure: <key name="python enable sandbox" type="bool">0</key>
+            sandbox_enabled = True
+            for key_elem in root.findall(f".//key[@name='{VRED_PREF_KEY_PYTHON_SANDBOX}']"):
+                if key_elem.text == "0":
+                    return False
+            return sandbox_enabled
+
+        except (ET.ParseError, OSError) as e:
+            logger.debug("Failed to read VRED preferences.xml: %s", e)
+            return True
+
+    def _confirm_override_update(self) -> bool:
+        """Ask user for confirmation to update existing VRED preferences override."""
+        try:
+            response = input(
+                "Existing VRED preferences override detected. "
+                "Do you want to update it with Deadline Cloud settings? (y/N): "
+            )
+            return response.lower().startswith("y")
+        except (EOFError, KeyboardInterrupt):
+            return False
+
+    def _create_merged_preferences_xml(self, existing_override: Optional[str]) -> str:
+        """
+        Create merged preferences XML that preserves all existing keys while updating
+        python sandbox and script settings for Deadline Cloud.
+        """
+        # Get merged script content
+        script_base64 = self._get_merged_script_base64(existing_override)
+
+        # Start with base XML structure
+        root = ET.Element("message")
+        root.set("id", "0")
+        root.set("type", "VRED")
+        root.set("version", "0.80000001")
+
+        # Copy all existing keys from override file if it exists
+        if existing_override:
+            existing_path = Path(existing_override)
+            if existing_path.exists():
+                try:
+                    existing_tree = ET.parse(existing_path)
+                    existing_root = existing_tree.getroot()
+
+                    # Copy all keys except the ones we'll override
+                    for key_elem in existing_root.findall(".//key"):
+                        key_name = key_elem.get("name")
+                        if key_name not in [
+                            VRED_PREF_KEY_PYTHON_SANDBOX,
+                            VRED_PREF_KEY_PYTOHN_SCRIPT,
+                        ]:
+                            root.append(key_elem)
+                except (ET.ParseError, OSError) as e:
+                    logger.debug("Failed to parse existing override XML: %s", e)
+
+        # Add/update Deadline Cloud specific keys
+        sandbox_key = ET.SubElement(root, "key")
+        sandbox_key.set("name", VRED_PREF_KEY_PYTHON_SANDBOX)
+        sandbox_key.set("type", "bool")
+        sandbox_key.text = "0"
+
+        script_key = ET.SubElement(root, "key")
+        script_key.set("name", VRED_PREF_KEY_PYTOHN_SCRIPT)
```

Comment:
Looks like a typo in a few spots, we should do a search/replace for this
```suggestion
        script_key.set("name", VRED_PREF_KEY_PYTHON_SCRIPT)
```

Reply:
Thank you for catching that!